modules/ROOT/pages/kafka/kafka-connector-design-center.adoc

```diff
@@ -15,118 +14,200 @@ For information on field values, see the xref:kafka/kafka-connector-reference.ad
 . Click the name of the trigger card.
-. If you are using the Kafka connector as an input source trigger, search for the connector; otherwise, search for HTTP or Scheduler.
-. Complete the fields with the values you want to configure for your trigger.
+. If you are using the Kafka connector as a trigger, search for the connector; otherwise, search for HTTP or Scheduler.
+. Select the source. +
+Kafka Connector provides the *Batch message listener* and *Message listener* as sources.
+. To configure the *Consumer*, set the field values:
 
-Design Center automatically saves changes you make in a session.
-
-The Kafka connector provides the *Message Consumer* operation as an input source.
-
-To configure the *Message Consumer* operation, set the field values:
-
-[%header,cols="30s,70a"]
-|===
-| Name | Description
-| Topic |  Name of the topic to consume messages from.
-| Connection |
-
-* <<basiccon>>
-* <<sslcon>>
-* <<kerbcon>>
-* <<kerbssl>>
-| Partition Offsets | List of offsets to configure. For each element in the list, specify a partition index and offset. 
-| Primary Node Only | Whether this source should execute only on the primary node in a cluster.
-|===
-
-To configure a consumer (trigger) for all configurations, set:
-
-* Consumer Partitions - The number of partitions to use for the consumer.
-* Group Id - A unique string that identifies the consumer group this consumer belongs to.
-
-[[basiccon]]
-=== Kafka Basic Consumer Connection
-
-[%header,cols="30s,70a"]
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
-| Name | Description
-|Bootstap Servers | Comma-separated host-port pairs used for establishing the initial connection to the Kafka cluster. This is the same as the `bootstrap.servers` value you must provide to Kafka clients (producer or consumer).
-| Additional Properties | Additional properties as key-value pairs that you need for your connection. Here you can put whatever property Kafka supports.
+| Name | Type | Description | Default Value | Required
+|Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
+| Default Acknowledgment Mode a| Enumeration, one of:
+
+** AUTO
+** MANUAL
+** IMMEDIATE
+** DUPS_OK a|  Defines the way that the Kafka broker instance is notified of the consumption of messages. 
+
+* AUTO: Messages are committed only if the flow finishes successfully. 
+* MANUAL: The user must commit manually through the Commit operation. 
+* IMMEDIATE: Mule automatically commits the messages upon reception and before triggering the flow. 
+* DUPS_OK: Same as the MANUAL mode, but the commit is made asynchronously, which can lead to duplicate records. | `AUTO` | 
+| Default Listener Poll Timeout a| Number |  The time, in time units, spent waiting to do a poll if data is not available in the buffer (fetched). If no value is set, returns immediately with any records that are available currently in the buffer, else returns empty. Must not be negative. |  `100` | 
```

Comment:
Need a subject for the second sentence. "If no value is set, the connection request??? returns immediately..."